bin/ci/deploy-dockerhub.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# THIS IS MEANT TO BE RUN BY CI
+
+set -e
```

Comment:
from https://github.com/mozilla-services/syncserver/pull/117#discussion_r186092204, should we add `-x` here?

Reply:
Done